asabiris/formatter/jinja/service.py

```diff
@@ -72,7 +72,14 @@ async def format(self, template_path, template_params):
 		# Load the template
 		template_io = await self.App.LibraryService.read(template_path)
 		if template_io is None:
-			raise PathError("Template '{}' not found".format(template_path))
+			raise ASABIrisError(
+				ErrorCode.TEMPLATE_NOT_FOUND,
+				tech_message="Failed to render. Reason : Template {} does not exist".format(template_path),
+				error_i18n_key="Failed to render. Reason : Template '{{incorrect_path}}' does not exist",
```

Comment:
I'll probably just keep the part "Template '{{incorrect_path}}' does not exist" - shorter and more user friendly? It will go on UI.
`Reason :  ` - extra spaces?